webapp/src/features/intersections/notifications/notifications-table.tsx

```diff
@@ -147,105 +150,90 @@ export const NotificationsTable = (props: { simple: Boolean }) => {
 
   return (
     <>
-      <Container maxWidth={false}>
-        {!simple && (
-          <>
-            <Box
-              sx={{
-                alignItems: 'center',
-                display: 'flex',
-                justifyContent: 'space-between',
-                flexWrap: 'wrap',
-                m: -1,
-              }}
-            >
-              <Grid2 container justifyContent="space-between" spacing={3}>
-                <Grid2>
-                  <Typography sx={{ m: 1 }} variant="h4" color="text.secondary">
-                    Notifications
-                  </Typography>
-                </Grid2>
-              </Grid2>
-              <Box
-                sx={{
-                  m: -1,
-                  mt: 3,
-                }}
-              ></Box>
-            </Box>
-            <Box
-              sx={{
-                m: -1,
-                mt: 3,
-                mb: 3,
-              }}
-            >
-              <Button
-                color="primary"
-                variant="contained"
-                onClick={updateNotifications}
-                startIcon={<RefreshIcon fontSize="small" />}
-                sx={{ m: 1 }}
-              >
-                Refresh
-              </Button>
-            </Box>
-          </>
-        )}
+      <Container
+        maxWidth={false}
+        sx={{
+          paddingLeft: '0px !important',
```

Comment:
Is this !important necessary? This can cause unintended side effects in the future.

Reply:
Nope - this has been removed.